app/views/static/datasets_terms.html.erb

```diff
@@ -2,7 +2,7 @@
   <div class="row">
     <div class="col-md-9">
       <h1><strong>Podmienky používania datasetov</strong></h1>
-      <p>Platné od 1.5.2018</p>
+      <p>Platné od 27.8.2018</p>
```

Comment:
Prechodne obdobie pred zmenou je minimalne 14 dni od zverejnenia cize to treba posunut povedzme na 15.9.

Reply:
Upravene.